lendingbot.py

```diff
@@ -94,6 +95,8 @@
                 exit(1)
             elif 'timed out' in ex.message:
                 print "Timed out, will retry in " + str(Lending.get_sleep_time()) + "sec"
+            elif isinstance(ex, BadStatusLine):
```

Comment:
It could but then we would need to duplicate the entire error logic that is in `except Exception:` already.